sdk/schemaregistry/schema-registry/sample.env

```
@@ -0,0 +1,17 @@
+# Used in most samples. Retrieve these values from a Cognitive Services instance
+# in the Azure Portal.
+ENDPOINT="Your Endpoint URL"
+API_KEY="Your API key"
```

We usually would like to preface service specific env vars with the service name to differentiate from similar env vars of other services.
And API_KEY is not applicable to SchemaRegistry.

```suggestion
SCHEMA_REGISTRY_ENDPOINT="Your Endpoint URL"
```